### ISSUE TITLE
Picker similar as IASKDatePickerSpecifier

### ISSUE THREAD

**Issue by cujo30227** [futuretap/InAppSettingsKit#432]
Is there currently a possibility to display a picker view, animating below the cell, which lists custom values. I.e. similar as the IASKDatePickerSpecifier but then without dates but with other values (example "1 Apple", "2 Apples" etc)

**Comment by futuretap** [comment]
No. Your best bet are radio elements or a multivalue element (which opens a child vc).
